packages/geo/src/lib/feature/feature-details/feature-details.component.ts

```diff
@@ -87,13 +91,15 @@ export class FeatureDetailsComponent {
   filterFeatureProperties(feature) {
     const allowedFieldsAndAlias = feature.meta ? feature.meta.alias : undefined;
     const properties = {};
+    const offlineButtonState = this.map.offlineButtonState;
 
     if (allowedFieldsAndAlias) {
       Object.keys(allowedFieldsAndAlias).forEach(field => {
         properties[allowedFieldsAndAlias[field]] = feature.properties[field];
       });
       return properties;
     } else {
+      if (!offlineButtonState) {
```

Comment:
Would be clearer if the "if" was with the "else" just above. I seem to have an unnecessary level.